edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -0,0 +1,95 @@
+var edx = edx || {};
+
+(function($) {
+  'use strict';
+
+  var actionToMessageTypesMap = {
+    'submit': {
+      promptEventName: 'endExamAttempt',
+      responseEventName: 'examAttemptEnded'
+    },
+    'start': {
+      promptEventName: 'startExamAttempt',
+      responseEventName: 'examAttemptStarted'
+    }
+  };
+
+  function workerPromiseForEventNames(eventNames) {
+    return function() {
+      var proctoringBackendWorker = new Worker(edx.courseware.proctored_exam.configuredWorkerURL);
+      return new Promise(function(resolve) {
```

Comment:
`(resolve) =>`

Reply:
this code doesn't run through babel, and I'm a bit anxious about browser compatibility